src/main/java/uk/gov/digital/ho/hocs/document/routes/MalwareCheckConsumer.java

```diff
@@ -85,16 +85,20 @@ public void configure() {
                     .process(generateDocumentConversionRequest())
                     .process(transferMDCToHeaders())
                     .to(toQueue)
+                    .endChoice()
                 .otherwise()
                 .when(HttpProcessors.validateHttpResponse)
                     .log(LoggingLevel.WARN, "Malware found")
                     .setProperty("status", simple(DocumentStatus.FAILED_VIRUS.toString()))
                     .process(generateDocumentUpdateRequest())
                     .process(transferMDCToHeaders())
                     .to(documentServiceQueueName)
+                    .endChoice()
                 .otherwise()
+                    .log(LoggingLevel.ERROR, "Malware check failed, response: ${body}")
```

Comment:
Should we add the following for when a generic failure occurs? 

`.setProperty("status", simple(DocumentStatus.FAILED_MALWARE_SCAN.toString()))`

Reply:
if we want to update the database yes but I think we also want to put it on the DLQ ourselves and receipt the message here rather than throwing an exception and letting AWS do it

Reply:
Yes, this isn't a code path where we want to retry

Reply:
I potentially disagree, this means an issue with ClamAV that could be due to load. Some instances of this would be worth retrying, such as when shared resources used are lower..?